core/src/main/java/org/jboss/sbomer/core/features/sbom/utils/PurlSanitizer.java

```diff
@@ -50,7 +50,7 @@ public static String sanitizePurl(String purl) {
             return parsedPurl.canonicalize();
         } catch (MalformedPackageURLException e) {
             // If parsing fails, proceed to manual sanitization
-            log.error("Malformed PURL detected, attempting to sanitize: {}", e.getMessage());
+            log.error("Malformed PURL detected, attempting to sanitize: {}", purl, e.getMessage());
```

Comment:
You now miss `{}` for the exception message. Or did you mean thins toi include stacktrace?
```suggestion
            log.error("Malformed PURL detected, attempting to sanitize: " + purl, e);
```

Reply:
Ah thanks I meant to get rid of the error message and stacktrace entirely, I overlooked!